CONTRIBUTING.md

```diff
@@ -104,15 +110,106 @@ Basically, this command will run
 # [dash-dev]
 $ yarn test
 ```
+
 #### Step 5: Start dashboard server and preview
+
 In this step, we will run the dashboard server and see the live reloadable preview.
 
 You can use the che command `dashboard_dev_server` (command pallette > Run task > … or containers view)
+
 ```bash
 # [dashboard_dev_server]
 $ node_modules/.bin/gulp serve --server=<che_api_url>
 ```
 
+### Workspace Loader
+
+Workspace loader source code is located in [<this.repo>/workspace-loader](./workspace-loader/) folder.
+Here is the developer workflow if you want to contribute to it:
+
+#### Devfile for workspace loader development
+
+The devfile: [<this.repo>/workspace-loader/devfile.yaml](./workspace-loader/devfile.yaml)
+
+This section shows you how to setup a Che 7 environment to work on the che workspace loader, and how to use it.
+For the whole workflows, we will need a workspace with such containers:
+
+- Workspace Loader Dev container (a.k.a ws-loader-dev) is a all in one container for running commands such as build, test or start the workspace loader server.
+
+All containers have `/projects` folder mounted, which is shared among them.
```

Comment:
> All containers have `/projects` folder mounted, which is shared among them.

Actually not all containers have `/projects` volume. Only containers marked in the devfile with attribute:  `mountSources: true`

Reply:
done

Reply:
removed